### DISCUSSION TITLE
Time set to clock not metric

### DISCUSSION THREAD

**Discussion by boudiccaeceni** [iSoron/uhabits#1502] [Feature Requests]
Habits in the time category are currently set up as base ten numbers, so the right hand side goes up to 100, not 60. So it's impossible to enter time correctly. There doesn't seem to be any point in having the time category since it's like this.

**Comment by kmsadaf** [comment]
Exactly, I also posted a thread on the same thing
Time input would be so helpful, I hope devs respond

**Comment by herrbrixter** [comment]
Duplicate of #953
